### PR TITLE
implment add and remove functionality

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
## What does this PR do
In this PR me and my coding partner check the error about the add and make the changes.
## Description of the task to be completed
- [x] Implement add and remove functionality
- [x] add the data to local storage

 
## How should it be tested
Clone the repository
```
https://github.com/AhmedMohsen600/to-do-list.git
```
Navigate to the folder
```
cd to-do-list
```
git checkout add
```
#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,35 +1,93 @@
...
+  static displayTodos = (todos) => {
+    const ul = document.querySelector('ul');
+    if (!todos.length) ul.innerHTML = '';
+    let content = '';
+    todos.forEach((tod) => {
+      content += `
+        <li id=${tod.id} class="item">
+          <div class='group'>
+            <input class='bdan' name='checkbox' type="checkbox" />
+            <p contenteditable="true">${tod.desc}</p>
```
Comment (christianbiring1) [inline src/index.js:21]: [OPTIONAL]
- [ ] After editing a list item when the page refreshes, the newly edited item is not saved. the previous one is the one that remains on the UI. Kindly you may consider changing that.
`Hint: You may make it in such a way that when updating the item, Its value is also being updated synchronously in the localSotorage` :smile:

File: src/index.js [hunk 2]
```diff
@@ -1,35 +1,93 @@
...
+//   let todos = Store.getTodos();
+//   let check = true;
+//   const koko = document.querySelectorAll('.bdan');
+//   console.log(koko);
+
+//   todos.map((todo) => {
+//     check ? { ...todo, completed: true } : todo;
+//   });
+//   console.log(todos);
+// };
```
Comment (christianbiring1) [inline src/index.js]: - [ ] Kindly consider removing the commented block of code in your code base as it makes your codebase not being clean

File: src/index.js [hunk 3]
```diff
@@ -1,35 +1,93 @@
...
+    else todos = JSON.parse(localStorage.getItem('todos'));
+    return todos;
+  };
+
+  static addTodo = (todo) => {
+    const todos = Store.getTodos();
+    todos.push(todo);
+    localStorage.setItem('todos', JSON.stringify(todos));
+    UI.displayTodos(todos);
+  };
```
Comment (christianbiring1) [inline src/index.js:53]: [OPTIONAL]
- [ ] You may also consider adding the list in such a way that when the user hit the `Enter` keyboard the item is added to the list :smile: 
`HINT:`
```
// Execute a function when the user presses a key on the keyboard
input.addEventListener("keypress", function(event) {
  // If the user presses the "Enter" key on the keyboard
  if (event.key === "Enter") {
    // Cancel the default action, if needed
    event.preventDefault();
    // Trigger the button element with a click
    document.getElementById("myBtn").click();
  }
});
```

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @AhmedMohsen600 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] The user can add and remove the item :+1: 

## Required Changes ♻️
- [x] `Kindly make sure the commented block of code is deleted in your code base`
- [x] `Also, make sure when the CLEAR ALL COMPLETED is clicked, all the checked tasks are deleted as it is shown in the video.`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (AhmedMohsen600) [reply src/index.js:53]: @christianbiring1  Thank you for your review. I have added this comment code cuz I'm going to use it on today's task

**Review by sajid-munawar — APPROVED** [review]
### Hi @AhmedMohsen600 , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**